stubs/parsimonious/parsimonious/nodes.pyi

```diff
@@ -0,0 +1,40 @@
+from re import Match
+from typing import Any, Callable, Iterator, NoReturn, TypeVar
+from typing_extensions import Self
+
+from parsimonious.exceptions import UndefinedLabel as UndefinedLabel, VisitationError as VisitationError
+from parsimonious.expressions import Expression as Expression
+from parsimonious.grammar import Grammar as Grammar
+
+class Node:
+    expr: Expression
+    full_text: str
+    start: int
+    end: int
+    children: list[Node]
+    def __init__(self, expr: Expression, full_text: str, start: int, end: int, children: list[Node] | None = ...) -> None: ...
+    @property
+    def expr_name(self) -> str: ...
+    def __iter__(self) -> Iterator[Node]: ...
+    @property
+    def text(self) -> str: ...
+    def prettily(self, error: Node | None = ...) -> str: ...
+
+class RegexNode(Node):
+    match: Match[str]
+
+class RuleDecoratorMeta(type):
+    def __new__(metaclass: type[Self], name: str, bases: tuple[type, ...], namespace: dict[str, Any]) -> Self: ...
+
+class NodeVisitor(type):
+    grammar: Grammar | Any
+    unwrapped_exceptions: tuple[Exception]
```

Comment:
```suggestion
    unwrapped_exceptions: tuple[type[Exception], ...]
```
`tuple[T]` means a one-tuple. And I suspect this holds exception classes, not instances.

Reply:
You're correct thanks!